euphony/src/main/java/co/euphony/tx/EuTxManager.kt

```diff
@@ -0,0 +1,173 @@
+package co.euphony.tx
+
+import android.media.AudioAttributes
+import android.media.AudioFormat
+import android.media.AudioManager
+import android.media.AudioTrack
+import android.os.Build
+import android.util.Log
+import co.euphony.common.Constants
+import co.euphony.common.EuNativeConnector
+import co.euphony.util.EuOption.ModeType
+import kotlinx.coroutines.CoroutineScope
+import kotlinx.coroutines.Dispatchers
+import kotlinx.coroutines.delay
+import kotlinx.coroutines.launch
+
+class EuTxManager {
+    private val txCore: EuNativeConnector = EuNativeConnector.getInstance()
+    private var audioTrack: AudioTrack? = null
+    private var playerEngineType: PlayerEngine? = null
+    var modeType: ModeType? = null
+        set(value) {
+            txCore.setMode(value)
+            field = value
+        }
+    var code: String?
+        get() = txCore.code
+        set(data) {
+            txCore.code = data
+        }
+
+    enum class EuPIDuration {
+        LENGTH_SHORT, LENGTH_LONG, LENGTH_FOREVER
+    }
+
+    enum class PlayerEngine {
+        ANDROID_DEFAULT_ENGINE, EUPHONY_NATIVE_ENGINE
+    }
+
+    fun callEuPI(freq: Double, duration: EuPIDuration): Constants.Result {
+        modeType = ModeType.EUPI
+        txCore.setAudioFrequency(freq)
+        val res = txCore.tx_start()
+        if (duration != EuPIDuration.LENGTH_FOREVER) {
+            CoroutineScope(Dispatchers.IO).launch {
+                delay(if (duration == EuPIDuration.LENGTH_SHORT) 200 else 500.toLong())
+                stop()
+            }
```

Comment:
> It would be nice to change `Handler` to `corountine`.
> 
> I wonder why 200 and 500 are specified as different types. left one is `Int` & right one is `Long` type. actually, delay's parameter type is `Long`. If the purpose is to specify the type, below is also fine :)

Thank you for your review! I didn't know the type was set like that, I changed it in this commit!
315c48c